### ISSUE TITLE
[Bug] Requires at least two episodes to render site

### ISSUE THREAD

**Issue by rajeshduggal** [mattstratton/castanet#153]
### Expected behavior
Should be able to render the site with less than two episodes.
### Actual behavior
With less than two episodes rendering fails with an error message in themes/partials/row.html:112
### Reproduction Steps
Starting from scratch, work through hugo quickstart getting started. Install castanet theme, run hugo server -D.  Notice the error. Run "hugo new episode/myepisode.md". Notice the error in the server output. Run "hugo new episode/myepisode2.md". Notice the website now renders.
### Operating System and Version
Windows 7 (cygwin)
### Web Browser and Version
n/a
### Hugo Version
hugo v.0.36..1

**Comment by mattstratton** [comment]
Good catch. I'll have to add some catch logic to not try to display the remaining episodes.

**Comment by rajeshduggal** [comment]
I found the bug while building our podcast http://khabardaarpodcast.com/  Fortunately, we decided to launch with an episode zero teaser at the same time as our episode one. (i.e. two episodes at launch).  So we jumped over the hurdle.  But, a fix will be needed for anyone else that wants to start using the theme.

**Comment by lmkirvan** [comment]
Only fails with row layout, just FYI.
